src/client/app/warehouses/warehouses.route.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('app.warehouses')
+    .run(appRun);
+
+  appRun.$inject = ['routerHelper'];
+  /* @ngInject */
+  function appRun(routerHelper) {
+    routerHelper.configureStates(getStates());
+  }
+
+  function getStates() {
+    return [
+      {
+        state: 'warehouses',
+        config: {
+          url: '/warehouses',
+          templateUrl: 'app/warehouses/warehouses.html',
+          controller: 'WarehousesController',
+          controllerAs: 'vm',
+          title: 'warehouses',
+          settings: {
+            nav: 3,
+            content: '<i class="fa fa-dashboard"></i>Warehouses'
```

Comment:
Do you know what is `<i class="fa fa-dashboard"></i>` and why fontawesome does not work here? :) can you please fix?
(just use this solution https://github.com/ITsvetkoFF/Kv-013/pull/148/files)